collector/fixtures/e2e-output.txt

```diff
@@ -1451,6 +1565,102 @@ node_netstat_Icmp_OutTimestampReps 0
 # HELP node_netstat_Icmp_OutTimestamps Protocol Icmp statistic OutTimestamps.
 # TYPE node_netstat_Icmp_OutTimestamps untyped
 node_netstat_Icmp_OutTimestamps 0
+# HELP node_netstat_Ip6_FragCreates Protocol Ip6 statistic FragCreates.
+# TYPE node_netstat_Ip6_FragCreates untyped
+node_netstat_Ip6_FragCreates 0
+# HELP node_netstat_Ip6_FragFails Protocol Ip6 statistic FragFails.
+# TYPE node_netstat_Ip6_FragFails untyped
+node_netstat_Ip6_FragFails 0
+# HELP node_netstat_Ip6_FragOKs Protocol Ip6 statistic FragOKs.
+# TYPE node_netstat_Ip6_FragOKs untyped
+node_netstat_Ip6_FragOKs 0
+# HELP node_netstat_Ip6_InAddrErrors Protocol Ip6 statistic InAddrErrors.
+# TYPE node_netstat_Ip6_InAddrErrors untyped
+node_netstat_Ip6_InAddrErrors 0
+# HELP node_netstat_Ip6_InBcastOctets Protocol Ip6 statistic InBcastOctets.
+# TYPE node_netstat_Ip6_InBcastOctets untyped
+node_netstat_Ip6_InBcastOctets 0
+# HELP node_netstat_Ip6_InCEPkts Protocol Ip6 statistic InCEPkts.
+# TYPE node_netstat_Ip6_InCEPkts untyped
+node_netstat_Ip6_InCEPkts 0
+# HELP node_netstat_Ip6_InDelivers Protocol Ip6 statistic InDelivers.
+# TYPE node_netstat_Ip6_InDelivers untyped
+node_netstat_Ip6_InDelivers 0
+# HELP node_netstat_Ip6_InDiscards Protocol Ip6 statistic InDiscards.
+# TYPE node_netstat_Ip6_InDiscards untyped
+node_netstat_Ip6_InDiscards 0
+# HELP node_netstat_Ip6_InECT0Pkts Protocol Ip6 statistic InECT0Pkts.
+# TYPE node_netstat_Ip6_InECT0Pkts untyped
+node_netstat_Ip6_InECT0Pkts 0
+# HELP node_netstat_Ip6_InECT1Pkts Protocol Ip6 statistic InECT1Pkts.
+# TYPE node_netstat_Ip6_InECT1Pkts untyped
+node_netstat_Ip6_InECT1Pkts 0
+# HELP node_netstat_Ip6_InHdrErrors Protocol Ip6 statistic InHdrErrors.
```

Comment:
The other source files are formatted differently. Most of these are obscure kernel stats that if you're lucky are mentioned on LKML somewhere, so keeping the original names increases the chances the users can find what little information there is about these stats.